### PR TITLE
Added technical-paper template

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
Hi Ron,
+ The structure of the template is based on "recommendations" template.
+ I'm not sure which text part should be considered as boilerplate. So, in some cases, I included commented text and others normal text.

Cheers

### CODE REVIEW

File: technical-paper/sections/01-scope.adoc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+== Scope
+// [ADD TEXT]
```
Comment (ronaldtse) [inline technical-paper/sections/01-scope.adoc]: Uncomment please.

File: technical-paper/sections/03-terms.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
+== Terms and definitions
+
+=== Terms defined elsewhere
```
Comment (ronaldtse) [inline technical-paper/sections/03-terms.adoc:4]: Convert all tabs into spaces.

File: technical-paper/sections/a1-annex.adoc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+[[annexA]]
+[appendix,obligation=normative]
+== Annex Title
+// [ADD TEXT.]
```
Comment (ronaldtse) [inline technical-paper/sections/a1-annex.adoc]: Uncomment please.

File: technical-paper/sections/05-contents.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+== Specific contents: could be introduction or something more substantive
+// [ADD TEXT.]
```
Comment (ronaldtse) [inline technical-paper/sections/05-contents.adoc]: Uncomment please.

File: technical-paper/sections/05-contents.adoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+.Example of a table
+[cols="2*",options="header"]
+|===
+|Number |Observation
+
+|1 |Sample table
+|2 |Table header (1st row) uses style Table_head
+|3 |Other table rows use style Table_text
+|4 |Heading with table number uses style "Table_No & title"
+|===
```
Comment (ronaldtse) [inline technical-paper/sections/05-contents.adoc:15]: Add empty line before table end (otherwise it is considered the table "footer")

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: technical-paper/sections/01-scope.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply technical-paper/sections/01-scope.adoc]: Ok, change applied.

File: technical-paper/sections/03-terms.adoc (hunk 2 above)
Comment (manuelfuenmayor) [reply technical-paper/sections/03-terms.adoc:4]: I don't see tabs, just empty lines.

File: technical-paper/sections/a1-annex.adoc (hunk 3 above)
Comment (manuelfuenmayor) [reply technical-paper/sections/a1-annex.adoc]: Change applied.

File: technical-paper/sections/05-contents.adoc (hunk 4 above)
Comment (manuelfuenmayor) [reply technical-paper/sections/05-contents.adoc]: Ok, change applied.

File: technical-paper/sections/05-contents.adoc (hunk 5 above)
Comment (manuelfuenmayor) [reply technical-paper/sections/05-contents.adoc:15]: Change applied.

**Pull request merged**